.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,7 +2,7 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: 'bug, priority: undecided'
+labels: 'bug'
```

Comment:
No need for the `priority: undecided`? I can see its usefulness for a PM.

Reply:
We have no priority labels. At this level priority doesn't exist for us.